### PR TITLE
6ticket single article

### PR DESCRIPTION
**Pull request by aDiallo21**

### CODE REVIEW

**Review by Augs0 — CHANGES_REQUESTED** [review]
This is almost there! Could you implement a 404 page for users to see when we go to a non-existent article? I'm also not sure why all your components are stored inside their own folders - this is not common practice

**Comment by aDiallo21** [comment]
Ok thanks for the feedback. Should I have a folder then with all the css files instead of a component folder with jsx and its css file

File: src/components/SingleArticle.jsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  const [error, setError] = useState(false);
+  const { article_id } = useParams();
+
+  useEffect(() => {
+    getArticleById(article_id)
+      .then((singleArticle) => {
+        setArticle(singleArticle);
+      })
+      .catch((err) => {
+        console.log(err, "ERROR");
```
Comment (yalicee) [inline src/components/SingleArticle.jsx:18]: consider removing console.log? I feel like this is personal preference since it is in the catch block

**Review by yalicee — APPROVED** [review]
Looks good to me :-)

**Pull request merged**